en/meta_attributes.rst

```diff
@@ -0,0 +1,98 @@
+	AT "http://models.geo.admin.ch/ASTRA/"
+	VERSION "2017-02-08"  =
+	  IMPORTS CatalogueObjects_V1,Units;
+	  !!@ topicInformation="Route-Type"
+	  TOPIC TypeOfRouteCatalogue
+	  EXTENDS CatalogueObjects_V1.Catalogues =
+	    !!@ dispExpression="type||' sometext'"
+	    CLASS TypeOfRoute
+	    EXTENDS CatalogueObjects_V1.Catalogues.Item =
+
+``furtherInformation`` is referenced to ``ExceptionalLoadsCatalogues_V1``, ``topicInformation`` to ``TypeOfRouteCatalogue`` and ``dispExpression`` to ``TypeOfRoute``.
+
+For more complex usage see the `specification of the Verein eCH <https://www.ech.ch/alfresco/s/ech/download?nodeid=788eb38a-bf2b-4f3d-96a8-addc37bba41f>`_
+
+Meta Attributes in the Database
+------------------------------------------
+
+When importing the data from the Interlis file to the DB, with ili2db integrated in the Project Generator, the Meta Attributes are stored in the table **t_ili2db_meta_attrs**:
+
+===============================================================  ==================  ===========================
+ilielement                                                       attr_name           attr_value
+===============================================================  ==================  ===========================
+ExceptionalLoadsCatalogues_V1                                    furtherInformation  https://www.astra.admin.ch/
+ExceptionalLoadsCatalogues_V1.TypeOfRouteCatalogue               topicInformation    Route-Type
+ExceptionalLoadsCatalogues_V1.TypeOfRouteCatalogue.TypeOfRoute   dispExpression      type||' sometext'
+===============================================================  ==================  ===========================
+
+Project Generator Specific Attributes
+================================================
+
+Some attributes the Project Generator can handle as properties in the QGIS project.
+
+List of specific Attributes
+------------------------------------------
+
+**dispExpression**
+Used as the display expression of a Relation Reference Widget. 
+
+|relation_reference|
+
+**not more specific attributes yet**
+
+
+Extra Meta Attributes File
+================================
+
+In these external files the meta attributes can be stored instead of having them directly in the Interlis files. 
+
+These files are written in the TOML and have the filename extension ``.toml``
+
+You can select the extra meta attribute files in the **Advanced Options**. This configuration is stored for the model. Means when you reselect the model later again, the file is still referenced. This information will be displayed on the main dialog of the Project Generator.
+
+In the background the ili2pg writes the attributes from the external meta attribute file first to the Interlis file and then to the PostGIS or GeoPackage storage, where the Project Generator can use them to build the QGIS project.
+
+**Example**
+
+::
+
+	["ExceptionalLoadsCatalogues_V1.TypeOfRouteCatalogue.TypeOfRoute"]
+	dispExpression="type||' sometext'"
```

Comment:
In general I use `CONCAT(attribute, ' text')` (rather than `||`) because

`CONCAT(NULL, 'text')` => `text` whereas `NULL || 'text'` => `NULL` which is confusing (and hard to debug).

For best practice, I'd probably make some examples

Simple attribute:

```
["ExceptionalLoadsCatalogues_V1.TypeOfRouteCatalogue.TypeOfRoute"]
dispExpression="type"
```

More Complex:
```
["Topic.BaseClass.Class"]
dispExpression="CONCAT(\"length\", 'm')"
```

Reply:
I did it like this, because it shows the user the possibilities with an easy to read example. 
But I can change it...

Reply:
Yes, I know the `||` approach is very intuitive and readable. If it wasn't for these hidden gotchas...